src/modules/RA/DataLayer/components/tabs/PopupTab/PopupFielSelect.js

```diff
@@ -23,17 +23,27 @@ const FieldSelect = ({
         || !popupfields.find(({ sourceFieldId }) => f.sourceFieldId === sourceFieldId))
     : fields), [fields, popupfields, selected]);
 
+  const validateLabel = React.useCallback(v => (v ? undefined : 'datalayer.form.popup.error.required'), []);
+
+
   return (
     <>
       {!selectable && (
-      <Field name={`${path}.label]`}>
-        {({ input: { value, onChange: onValueChange } }) => (
+      <Field
+        name={`${path}.label]`}
+        validate={validateLabel}
+        parse={v => v}
```

Comment:
Why do you need this parse ?

Reply:
If I understand correctly, `react-final-form` "remove" undefined key from form (empty string `""` are set as undefined). With the layerfields/sourcefield merge, it replace the `label` with the `name`. The parse function will force empty string into the form allowing a correct  validation